### DISCUSSION TITLE
how i can convert to an executable program?

### DISCUSSION THREAD

**Discussion by Beat25cm** [TomSchimansky/CustomTkinter#1228] [General]
![image](https://user-images.githubusercontent.com/59614073/218631851-a8544323-4d5c-43c1-bc75-d686cd822232.png)

**Comment by Akascape** [comment]
@Beat25cm Copy the customtkinter folder in the same directory where your program is present, then add that folder as `additional folder` under pyinstaller.

Here is a full guide:
https://github.com/TomSchimansky/CustomTkinter/discussions/939

**Reply by Beat25cm** [reply]
thanks for helping @Akascape , 
is there any way to convert it into a single file?

**Reply by Akascape** [reply]
@Beat25cm Everything is given in the guide!

**Reply by Wolf-SO** [reply]
@Akascape You mean [wiki/Packaging](../wiki/Packaging)?

**Reply by Akascape** [reply]
@Wolf-SO I mean this improved discussion about packaging: #939

**Reply by Wolf-SO** [reply]
@Akascape What is the correct way to get this incorporated into the wiki page?

**Reply by Akascape** [reply]
@Wolf-SO only **TomSchimansky** can update the wiki page
